helms/monitoring/templates/grafana/datasource.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    grafana_datasource: "1"
+    monitoring: prometheus
+  name: prometheus-grafana-datasource
+data:
+  datasource.yaml: |-
+    apiVersion: 1
+    datasources:
+    - name: Prometheus
+      type: prometheus
+      url: http://prometheus-prometheus:9090/
```

Comment:
Might we can remove hardcode here?

Reply:
A user must not modify this URL because Prometheus has deployed in this chart too. And the Prometheus service is immutable.

Reply:
Fixed